CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ## [Unreleased]
 
-- No changes yet.
+- Remove `buf beta registry template` commands.
+- Move `buf alpha plugin` commands to `buf beta registry plugin` and remove old plugin implementation.
```

Comment:
Maybe be more specific and state that the following old plugin commands are removed:
* `buf beta registry plugin create`
* `buf beta registry plugin deprecate`
* `buf beta registry plugin list`
* `buf beta registry plugin undeprecate`
* `buf beta registry plugin version`

and that these commands now work with remote plugins:
* `buf beta registry plugin delete`
* `buf beta registry plugin push`

Is it worth also linking to docs somewhere re: custom plugins or the alpha -> remote plugins/packages migration pages?